common-ethereum/src/test/java/nl/tudelft/trustchain/common/ethereum/WalletPasswordUtilsTest.kt

```diff
@@ -0,0 +1,23 @@
+package nl.tudelft.trustchain.common.ethereum
+
+import nl.tudelft.trustchain.common.ethereum.utils.generateWalletPassword
+import org.junit.Test
+import java.util.*
+
+private const val PASSWORD_REGEX = "[a-zA-Z0-9!]+";
+
+class WalletPasswordUtilsTest {
+
+    @Test
+    fun generatedWalletPassword_isCorrect() {
+        // given
+        val random = Random(0)
```

Comment:
Java Random is not considered safe for password randomness.

Reply:
It's a unit test, so it does not matter that much. Nevertheless, I've replaced it with a `SecureRandom`.